server/lib/wikidata/aliases.js

```diff
@@ -110,6 +111,22 @@ const wikidataOnlyP31Values = {
     'wd:Q49773', // social movement
     'wd:Q2738074', // political movement
   ],
+  languages: [
+    'wd:Q34770', // language
+    'wd:Q1288568', // modern language
+    'wd:Q38058796', // extinct language
+    'wd:Q45762', // dead language
+    'wd:Q33384', // dialect
+    'wd:Q33742', // natural language
+    'wd:Q34228', // sign language
+    'wd:Q436240', // ancient language
+    'wd:Q2315359', // historical language
+    'wd:Q315', // language
+    'wd:Q33215', // constructed language
+    'wd:Q838296', // international auxiliary language
+    'wd:Q2519134', // engineered language
+    'wd:Q17376908', // languoid
```

Comment:
Based on a different approach to [query languages]() (starting from P424 wikimedia language code) I could isolate slightly more items which seem to be relevant for this list.

This is the result of an original intention of having only a sparql query to define languages (such as in  [#670](https://github.com/inventaire/inventaire/pull/670/files#diff-64bac6e676260e99436e6dba5d314fe0d8546587a05895a3feacf8cf6cb8847b)) but heuristics got way too numerous to not have a good old alias solution (surprise! (no))

```suggestion
    'wd:Q315', // language
    'wd:Q33215', // constructed language
    'wd:Q33289', // creole language
    'wd:Q33384', // dialect
    'wd:Q33742', // natural language
    'wd:Q34228', // sign language
    'wd:Q34770', // language
    'wd:Q43091', // orthography
    'wd:Q45762', // dead language
    'wd:Q152559', // ISO 639-3 macrolanguage
    'wd:Q399495', // standard language
    'wd:Q436240', // ancient language
    'wd:Q838296', // international auxiliary language
    'wd:Q839470', // sacred language
    'wd:Q1149626', // written language
    'wd:Q1208380', // dialect group
    'wd:Q1288568', // modern language
    'wd:Q2315359', // historical language
    'wd:Q2519134', // engineered language
    'wd:Q17376908', // languoid
    'wd:Q38058796', // extinct language
    'wd:Q64362969', // language in script
```

added items
```
'wd:Q838296', // international auxiliary language
'wd:Q1149626', // written language
'wd:Q1208380', // dialect group
'wd:Q152559', // ISO 639-3 macrolanguage
'wd:Q33289', // creole language
'wd:Q399495', // standard language
'wd:Q43091', // orthography
'wd:Q64362969', // language in script
'wd:Q839470', // sacred language
```

Reply:
done in ab28e59cc